mails/themes/classic/core/backoffice_order.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends '@MailThemes/classic/components/layout.html.twig' %}
+
+{% block content %}
+<tr>
+  <td align="center" class="titleblock">
+    <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+      <span class="title">{{ 'Hi {firstname} {lastname},'|trans({}, 'EmailsBody', locale)|raw }}</span>
+    </font>
+  </td>
+</tr>
+<tr>
+  <td class="space_footer">&nbsp;</td>
+</tr>
+<tr>
+  <td class="box" style="border:1px solid #D6D4D4;">
+    <table class="table">
+      <tr>
+        <td width="10">&nbsp;</td>
+        <td>
+          <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+            <p style="border-bottom:1px solid #D6D4D4;">
+              {{ 'A new order has been generated on your behalf.'|trans({}, 'EmailsBody', locale)|raw }}
```

Comment:
An y reason to use raw here?

Reply:
These templates were generated by a script, based on the existing ones in the `ps_emailgenerator` module. I thought it was better to use raw everywhere as we don't know what content the translations can have depending on the language. And they may include html tags.
Do you see any reason why we shouldn't use raw?

Reply:
We can inject html in raw string.
https://twig.symfony.com/doc/2.x/filters/raw.html
It mark the value of all variables as safe, which is clearly false in our case. Maybe better to not use raw as much as possible.

Reply:
I updated the templates so that `raw` is not used when there are variables (unless it contains html code).